app/util/mapIconUtils.js

```diff
@@ -350,6 +363,38 @@ function getSmallStopIcon(type, radius, color) {
   });
 }
 
+/**
+ * Draw a badge icon on top of the icon.
+ */
+function drawStopStatusBadge(
+  tile,
+  x,
+  y,
+  iconWidth,
+  stopOutOfService,
+  noRoutesForStop,
+) {
+  const badgeSize = iconWidth * 0.75; // badge size is 75% of the icon size
+  const badgeImageId = stopOutOfService
+    ? `icon-icon_stop-temporarily-closed-badge`
+    : `icon-icon_stop-closed-badge`;
```

Comment:
Which mode does this stop icon represent (the sprite files does not show up in git)?
Another option to handle badges I'm currently exploring with rental stations, is having badges included in the original svg, and activate/color them via css vars. This would decouple the code from knowing where to place the badge.

Reply:
These are universal badges for any mode. Grey clock symbol / cross on red background.